grakn-engine/src/main/java/ai/grakn/engine/tasks/mock/MockBackgroundTask.java

```diff
@@ -88,12 +88,16 @@ public static void clearTasks() {
 
     @Override
     public final boolean start() {
-        Json json = configuration().json();
+        Json json = Json.read(configuration().configuration());
+        if(json.equals(Json.object())){
+            throw new RuntimeException("Invalid task config");
+        }
+
         Json id = json.at("id");
         if (id == null) {
             LOG.error("Missing id in {}: {}", this.getClass().getName(), json);
         }
-        this.id = TaskId.of(id.asString());
+        this.id = TaskId.generate();
```

Comment:
It looks like we ignore the ID from the JSON configuration here. Is that deliberate? I can't remember how any of this task stuff works anymore.

Reply:
The Task ID is kind of in limbo right now. It's not actually not used anywhere anymore properly. The old property tests use it in weird ways, hence the weird mock above.

I could get rid of the old property tests but I am not 100% sure yet if I should.

These odd mocks should die in another PR though. They probably made sense at some point but not anymore